Resources/Changelog/GoobChangelog.yml

```
@@ -11303,3 +11323,51 @@ Entries:
       message: Security got new melee toys to play with.
   id: 1242
   time: '2025-06-15T08:15:54.0000000+00:00'
+- author: pheenty
+  changes:
+    - type: Fix
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)